File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/HideNotificationViewFragment.java
package com.regula.facesample.items.customization.fragment;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;

import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 6.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class HideNotificationViewFragment extends FaceDefaultUiFragment {

    @Override
    protected TextView getNotificationView(@NonNull View view) {
        TextView textView = super.getNotificationView(view);
        textView.setVisibility(View.INVISIBLE);
        return textView;
    }

    @Override
    public void setNotificationStatusText(String s) {
        // ignore any new status
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/fragments/HideNotificationViewFragment.kt
package com.regula.facesamplekotlin.items.fragments

import android.view.View
import android.widget.TextView
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class HideNotificationViewFragment : FaceDefaultUiFragment() {

    override fun getNotificationView(view: View): TextView? {
        val textView: TextView = super.getNotificationView(view)
        textView.visibility = View.INVISIBLE
        return textView
    }

    override fun setNotificationStatusText(s: String?) {
        // ignore any new status
    }

}